Add FAMC.STAT UNCHALLENGED (#224)
FamilySearch/GEDCOM: specification/gedcom-03-datamodel.md

| `CHALLENGED` | Linking this child to this family is suspect, but the linkage has been neither proven nor disproven. |
| `DISPROVEN` | There has been a claim by some that this child belongs to this family, but the linkage has been disproven. |
| `PROVEN` | There has been a claim by some that this child does not belong to this family, but the linkage has been proven. |

> | `UNCHALLENGED` | No reason to suspect the link between this child and this family has been identified. `UNCHALLENGED` is implied if this structure is not present. |

Reviewer: no reason to suspect it's been identified sounds like it's in doubt, is that what is meant?  I might suggest

```suggestion
| `UNCHALLENGED` | There have been no known claims that the link between this child and this family is suspect. `UNCHALLENGED` is implied if this structure is not present. |
```

I would also ask about the stated meaning of PROVEN.   If the linkage has been proven, but there has _not_ been a claim that this child does not belong to this family, what status should be used?

Author: `PROVEN` was added with this wording in 5.5.1; not sure what the rationale was

Reviewer: Tracking via issue #225 and proposed PR #228

> | `UNCHALLENGED` | There have been no known claims that the link between this child and this family is suspect. `UNCHALLENGED` is implied if this structure is not present. |

Author: ```suggestion
| `UNCHALLENGED` | There have been no known claims that the link between this child and this family is suspect. |
```

Remove the "if not present" clause due to comments in pull request conversation thread
